coalib/results/Diff.py

```diff
@@ -20,11 +20,14 @@ def __init__(self, file_list=None,
         Creates an empty diff for the given file.
 
         :param file_list: The original (unmodified) file as a list of its
-                          lines.
+                          lines. It has to be provided if create is not set,
+                          else an error will be raised.
```

Comment:
I think this doc-comment change should be squashed with the commit introducing the error ;)